Form/Type/ChangePasswordFormType.php

```diff
@@ -22,23 +23,23 @@ class ChangePasswordFormType extends AbstractType
     private $class;
 
     /**
-     * @param string $class The User class name
+     * @param string|null $class The User class name
      */
-    public function __construct($class)
+    public function __construct($class = null)
     {
-        $this->class = $class;
+        $this->class = $class ?: UserInterface::class;
     }
 
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
-        $builder->add('current_password', 'password', array(
+        $builder->add('current_password', Type\PasswordType::class, array(
```

Comment:
This will only work on newer versions of Symfony. FOSUserBundle supports 2.3 LTS.